Engine/source/console/consoleObject.h

```diff
@@ -542,6 +545,19 @@ class AbstractClassRep : public ConsoleBaseType
 
 extern AbstractClassRep::FieldList sg_tempFieldList;
 
+inline AbstractClassRep* AbstractClassRep::getClassRepFromName(const char* className)
+{
+	AbstractClassRep* current = classLinkList;
+	while(current != NULL)
+	{
+		if(dStricmp(current->getClassName(),className) == 0)
```

Comment:
It really begs the question, what's the greater cost? A call to `insert` or a string comparison?